lib/power_action_utils.pm

```diff
@@ -49,20 +49,26 @@ $vnc_console get required variable 'SVIRT_VNC_CONSOLE' before assignment.
 =cut
 sub prepare_system_shutdown {
     # kill the ssh connection before triggering reboot
-    console('root-ssh')->kill_ssh if get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/;
-
-    if (check_var('ARCH', 's390x')) {
+    if (get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/) {
+      console('root-ssh')->kill_ssh if get_var('BACKEND', '') =~ /ipmi|spvm|pvm_hmc/;
+      record_info('prepare_system_shutdown: ssh connection is disconnected');
+    }
+    elsif (check_var('ARCH', 's390x')) {
         if (check_var('BACKEND', 's390x')) {
             # kill serial ssh connection (if it exists)
             eval { console('iucvconn')->kill_ssh unless get_var('BOOT_EXISTING_S390', ''); };
+            record_info('prepare_system_shutdown: ssh connection is disconnected');
```

Comment:
I suggest to not repeat the same string in info messages. Better make them distinguishable.